src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -145,23 +114,58 @@
 					@click="handleCreateConversation">
 					{{ t('spreed', 'Create conversation') }}
 				</NcButton>
-				<!-- Third page -->
-				<NcButton v-if="page===2 && (error || isPublic)"
-					ref="closeButton"
-					type="primary"
-					class="new-group-conversation__button"
-					@click="closeModal">
-					{{ t('spreed', 'Close') }}
-				</NcButton>
 			</div>
 		</NcModal>
+
+		<!-- Third page : this is the confirmation page-->
+		<NcModal v-else-if="page === 2"
+			:container="container"
+			@close="closeModal">
+			<NcEmptyContent>
+				<template #icon>
+					<Check v-if="!error && success && isPublic" :size="64" />
+					<div v-if="isLoading && !error && !success" class="icon-loading spinner" />
+					<AlertCircle v-if="error" :size="64" />
+				</template>
+
+				<template #description>
+					<p v-if="isLoading && !error && !success" class="confirmation__warning">
+						{{ t('spreed', 'Creating the conversation …') }}
+					</p>
+					<p v-if="!error && success && isPublic" class="confirmation__warning">
+						{{ t('spreed', 'All set, the conversation "{conversationName}" was created.', { conversationName }) }}
+					</p>
+					<p v-if="error" class="confirmation__warning">
+						{{ t('spreed', 'Error while creating the conversation') }}
+					</p>
+				</template>
+
+				<template #action>
+					<NcButton v-if="error || isPublic"
+						ref="closeButton"
+						type="primary"
```

Comment:
I chose them differently because : 
- `primary` : because it will change the route and redirect you to another conversation : seems like a big action , no ? 
- `tertiary `: because it will only copy a link to clipboard.

Reply:
No, I meant "Close" button =)

Secondary for "Copy link" seems fine, as well as primary for "Go to conversation"

Reply:
aaa oops, I misread which button you are referring to .